examples/datasource-logs/src/components/QueryEditor.tsx

```diff
@@ -12,18 +12,17 @@ export function QueryEditor({ query, onChange }: Props) {
   };
 
   const onQueryLimitChange = (event: ChangeEvent<HTMLInputElement>) => {
-    onChange({ ...query, limit: parseInt(event.target.value,10) });
+    onChange({ ...query, limit: parseInt(event.target.value, 10) });
   };
 
   return (
-    <div className="gf-form">
+    <HorizontalGroup spacing="none">
```

Comment:
I've a feeling we should be using `[<Stack />](https://developers.grafana.com/ui/latest/index.html?path=/story/general-layout-stack--direction-examples)` these days rather than these older layout components.

Reply:
Good shout 👍